docs/getting-started/shipping-some-logs.md

```diff
@@ -2,56 +2,142 @@
 title: "Shipping Some Logs"
 ---
 
-Now that you have a logsearch deployment running (albeit a small one), you can start shipping some logs into it.
-Whenever you are configuring a log shipper (regardless of deployment) there are a few important things you'll need to
-know...
+A Logsearch cluster with no logs is sad.  Lets make the local test cluster you just deployed happy by shipping it some logs.
 
- * protocol - whether to ship via syslog, relp, lumberjack, etc
- * endpoint - which hostname/IP and port to point to
- * security - some ingestors will require a TLS key/certificate to be configured.
+# Concepts
 
-For this guide, we'll use the simplest method: ship with the lumberjack protocol to our ingestor at `10.244.2.14:5043`
-with the TLS key that you can find in the example manifest. You can download logstash-forwarder from [gobuild.io][1] to
-get started. On Mac OS X you could temporarily download it by running...
+## Shipping
 
-    $ mkdir /tmp/logstash-forwarder
-    $ cd /tmp/logstash-forwarder
-    $ curl -LO http://gobuild3.qiniudn.com/github.com/elasticsearch/logstash-forwarder/branch-v-master/logstash-forwarder-darwin-amd64.zip 
-    $ unzip logstash-forwarder-darwin-amd64.zip 
-    $ ./logstash-forwarder -h
+We call the act of getting logs into the Logsearch cluster shipping.  Logs can come from may places; but most often they are in the form of a log file on a server's disk.
```

Comment:
I'd change 'getting logs' to 'ingesting logs', which is more widely used these days I think - i.e. we use 'shippping' rather than 'ingesting' (not sure whether we are consistent in this regard though ;)